conf/modules.config

```diff
@@ -47,12 +47,12 @@ process {
     //
     withName: '.*:FASTQ_FASTQC_UMITOOLS_FASTP:FASTP' {
         ext.args = [ "",
-            params.trim_fastq                  ? "" : "--disable_adapter_trimming",
-            params.clip_r1 > 0                 ? "--trim_front1 ${params.clip_r1}" : "", // Remove bp from the 5' end of read 1.
-            params.three_prime_clip_r1 > 0     ? "--trim_tail1 ${params.three_prime_clip_r1}" : "", // Remove bp from the 3' end of read 1 AFTER adapter/quality trimming has been performed.
-            params.fastp_min_length > 0        ? "-l ${params.fastp_min_length}" : "",
-            params.fastp_max_length > 0        ? "--max_len1 ${params.fastp_max_length}" : "",
-            params.three_prime_adapter == null ?  '' : "--adapter_sequence ${params.three_prime_adapter}"
+            params.trim_fastq                           ? "" : "--disable_adapter_trimming",
+            params.clip_r1 > 0                          ? "--trim_front1 ${params.clip_r1}" : "", // Remove bp from the 5' end of read 1.
+            params.three_prime_clip_r1 > 0              ? "--trim_tail1 ${params.three_prime_clip_r1}" : "", // Remove bp from the 3' end of read 1 AFTER adapter/quality trimming has been performed.
+            params.fastp_min_length > 0                 ? "-l ${params.fastp_min_length}" : "",
+            params.fastp_max_length > 0                 ? "--max_len1 ${params.fastp_max_length}" : "",
+            params.three_prime_adapter == "auto-detect" ?  "" : "--adapter_sequence ${params.three_prime_adapter}"
```

Comment:
Does this need updating in the enumation in `nextflow_schema.json`?

Reply:
Currently we do not have any enumation for it.